CMakeLists.txt

```diff
@@ -62,6 +62,10 @@ option(NRN_ENABLE_RX3D "Enable rx3d support" ${NRN_ENABLE_RX3D_DEFAULT})
 option(NRN_ENABLE_CORENEURON "Enable CoreNEURON support" ${NRN_ENABLE_CORENEURON_DEFAULT})
 option(NRN_ENABLE_BACKTRACE "Enable pretty-printed backtraces" ${NRN_ENABLE_BACKTRACE_DEFAULT})
 option(NRN_ENABLE_TESTS "Enable unit tests" ${NRN_ENABLE_TESTS_DEFAULT})
+option(NRN_ENABLE_MODEL_TESTS "Enable tests with detailed models specified" ${NRN_ENABLE_MODEL_TESTS_DEFAULT})
+# parse NRN_ENABLE_MODEL_TESTS as list
+string(REPLACE " " ";" NRN_ENABLE_MODEL_TESTS ${NRN_ENABLE_MODEL_TESTS})
```

Comment:
Is this needed? IIUC it enables `"-DNRN_ENABLE_MODEL_TESTS=a b"`, but naïvely it seems like `DNRN_ENABLE_MODEL_TESTS=a,b` is sufficient.

Reply:
Not really. It can be removed